pkg/core/native/interop.go

```diff
@@ -57,6 +64,10 @@ func OnPersist(ic *interop.Context) error {
 		return errors.New("onPersist must be trigered by system")
 	}
 	for _, c := range ic.Natives {
+		history := ic.Chain.GetConfig().GetNativeUpdateHistory(c.Metadata().Name)
```

Comment:
We could then avoid going to the chain here, just add some `IsActive(height uint32)` method.